meetings/2020-11-16.md

```diff
@@ -0,0 +1,13 @@
+# ASWF Python 3 Working Group Meeting - 2020/11/16
+## Attendees
+
+# Action Items
+
+# Agenda
+* Notetaker: Your name
+* Review pending action points
+	- [ ] Add [What’s new & what’s going away in 3.9](https://docs.python.org/3.9/whatsnew/3.9.html#removed) to repo notes **JT Nelson**
+	- [ ] Curate the guidelines docs **David Aguilar**
```

Comment:
Thanks for the reminder. What's the best way to do the curation in a way that's best discoverable/usable by the team?  I can speak to the intent and lineage of the guideline docs.

My sug would be that I can write an appendix section to the guidelines with the following:
- Similarities to the Python Futures documentation and the Conservative Python Porting Guide.
- Highlight where we diverge from the above guidelines, with rationale describing why.
- Highlight the additional guidance that is not covered by the above guides.

I think it would also be good to add an intro section where we describe the high-level goals of the guidelines with context around the intended use cases where it may and may not apply.  Eg. the guidance should make it clear that it's for Python code in the context of a studio environment where we want to maintain a single code base.

I'll have this ready for tomorrow.  Thanks Ashley!